stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamColors.kt

```diff
@@ -84,6 +88,8 @@ public data class StreamColors(
     public val imageBackgroundMediaGalleryPicker: Color,
     public val videoBackgroundMessageList: Color,
     public val videoBackgroundMediaGalleryPicker: Color,
+    public val showMoreOverlay: Color,
```

Comment:
This might not seem like it pulls its own weight, but we don't directly read a resource anywhere in our code base and I did not want to set a bad precedent and lock out users from having the ability to customize this, hence adding new properties.

Reply:
We hardcode literal white color in other places: unread count, cooldown indicator, etc.